yb-voyager/cmd/common.go

```diff
@@ -970,6 +970,39 @@ func storeTableListInMSR(tableList []sqlname.NameTuple) error {
 	return nil
 }
 
+var appChanges = []string{
+	INHERITANCE_ISSUE_REASON,
+	CONVERSION_ISSUE_REASON,
+	DEFERRABLE_CONSTRAINT_ISSUE,
+	fmt.Sprintf("%s - xml", UNSUPPORTED_DATATYPE),
+	fmt.Sprintf("%s - xid", UNSUPPORTED_DATATYPE),
+	UNSUPPORTED_EXTENSION_ISSUE, // will confirm this
+}
+
+// Migration complexity calculation from the conversion issues
+func getMigrationComplexity(sourceDBType string) string {
```

Comment:
let's pass in the schemaAnalysisReport to this function instead of relying on the global variable?